### PR TITLE
websocket: switch to nhooyr.io/websocket

### PR DESCRIPTION
**Pull request by psanford**
This allows for use from a wasm environment.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,5 +8,6 @@ require (
 	github.com/klauspost/compress v1.11.4
 	github.com/spf13/cobra v1.1.3
 	golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5
+	nhooyr.io/websocket v1.8.6
```
Comment (Jacalz) [inline go.mod:11]: Shouldn’t `github.com/gorilla/websocket v1.4.2 ` be removed from the imports now that this is added?

**Comment by Jacalz** [comment]
How does this compare performance wise to the old websocket library?

File: go.mod (hunk 1 above)
Comment (psanford) [reply go.mod:11]: Nope, the tests are still using `gorilla/websocket` for the stub rendezvous server

**Comment by psanford** [comment]
> How does this compare performance wise to the old websocket library?

The websocket library is only used to talk to the rendezvous server. I would expect any library performance differences to be negligible compared to the network latency involved in the actual file transfer part of the process.

File: go.mod (hunk 1 above)
Comment (Jacalz) [reply go.mod:11]: Is there any reason that those couldn't use this as well? Seems better to just use one in my opinion.

Comment (psanford) [reply go.mod:11]: The idea is we want to validate the new implementation against the existing tests. If the new implementation is correct we shouldn't need to make any adjustments to the tests.

When we are confident everything is correct we can consider switching the test to use the same library.

**Pull request closed without merging**